ocaml/doc/README.md

```diff
@@ -0,0 +1,11 @@
+# A note on generating locally the API reference
+
+Run `make doc`. The output is in `_build/ocaml/doc/markdown`.
+
+To view the markdown properly you will need to install `graphviz` so as to
+convert the dot files to images. To generate a pdf in addition, you will need to
+install `pandoc`.
```

Comment:
From what I see in the docs pdflatex is the engine option, but the thing to install is latex, the actual package name depending on the distro...